deployment/bicep/shared/shared.bicep

```diff
@@ -23,36 +23,36 @@ module appInsights './azmon.bicep' = {
   params: {
     location: location
     sharedResourceGroupResources : sharedResourceGroupResources
-
   }
 }
+
 output appInsightsConnectionString string = appInsights.outputs.appInsightsConnectionString
 output appInsightsName string = appInsights.outputs.appInsightsName
 output appInsightsId string = appInsights.outputs.appInsightsId
 output appInsightsInstrumentationKey string = appInsights.outputs.appInsightsInstrumentationKey
 
 module vm_devopswinvm './createvmwindows.bicep' = {
-  name: 'azdevopsvm'
+  name: 'vm-azdevops'
   scope: resourceGroup(resourceGroupName)
   params: {
     subnetId: agentSubnetId
     username: vmazdevopsUsername
     password: vmazdevopsPassword
-    vmName: 'azdevops-${sharedResourceGroupResources.vmSuffix}'
+    vmName: 'vm-ado-${sharedResourceGroupResources.vmSuffix}'
     azureDevOpsAccount: azureDevOpsAccount
     personalAccessToken: personalAccessToken
     deployAgent: true
   }
 }
  
 module vm_jumpboxwinvm './createvmwindows.bicep' = {
-  name: 'jumpboxwinvm'
+  name: 'vm-jumpbox'
   scope: resourceGroup(resourceGroupName)
   params: {
     subnetId: jumpboxSubnetId
     username: vmazdevopsUsername
     password: vmazdevopsPassword
-    vmName: 'jumpbox-${sharedResourceGroupResources.vmSuffix}'
+    vmName: 'vm-jmpbx-${sharedResourceGroupResources.vmSuffix}'
```

Comment:
Is there a character limit to spell `jmpbx` out? I'd probably just add the two vowels.